imageroot/actions/get-defaults/20readconfig

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import json
+import os
+import sys
+import agent
+import subprocess
+
+rdb = agent.redis_connect() # full read-only access on every key
+modules=[]
+accepted_timezone_list = []
+list_of_timezone_widget = []
+
+# we query about all mail server to use it inside the user interface
+for key in agent.list_service_providers(rdb,'imap','tcp'):
+    modules.append({
+        "name": key['module_id'],
+        "label": key['mail_hostname'],
+        "value": key['module_id'],
+    })
+# we query about all known timezone and we build a widget
+accepted_timezone_list = subprocess.check_output(['podman', 'run', '--rm', os.environ["WEBTOP_WEBAPP_IMAGE"], 'java', '-classpath', '/usr/share/webtop/', 'ListTimeZones'], text=True).splitlines()
+
+for value in accepted_timezone_list:
+    list_of_timezone_widget.append({
+        "name": value,
+        "label": value,
+        "value": value,
+    })
+
+config={
+        "mail_module_widget": modules,
```

Comment:
I prefer a more generic name, this API can be used also outside the scope a Web UI
```suggestion
        "mail_modules_id": modules,
```